src/tests/system/tests/test_gpo.py

```diff
@@ -0,0 +1,881 @@
+"""
+SSSD GPO Tests
+
+Switch user (su), and Remote (ssh) access is parameterized fixture for the tests. When SeRemoteInteractiveLogonRight is
+omitted from the policy, which are most tests cases, the value from SeInteractiveLogonRight is then copied to
+SeRemoteInteractiveLogonRight.
```

Comment:
Hi,

maybe you can add here that `ad.group("Domain Admins")` should always be allowed to make the framwork work as expected.

bye,
Sumit